website/docs/d/standards_resource_scores.html.markdown

```diff
@@ -0,0 +1,51 @@
+---
+layout: "pagerduty"
+page_title: "PagerDuty: pagerduty_standards_resource_scores"
+sidebar_current: "docs-pagerduty-datasource-standards-resource-scores"
+description: |-
+  Get information about the scores received for the standards associated to a
+  single resource.
+---
+
+# pagerduty\_standards\_resource\_scores
+
+Use this data source to get information about the [scores for the standards of a
+resource][1].
+
+## Example Usage
+
+```hcl
+data "pagerduty_standards_resource_scores" "scores" {
+    resource_type = "technical_services"
+    id = "P703E9Q"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `resource_type` - Type of the object the standards are associated to. Allowed values are `technical_service`.
```

Comment:
Please mind the typo in `technical_service`, shouldn't it be `technical_services`?